NumLib/DOF/LocalToGlobalIndexMap.h

```diff
@@ -174,6 +189,17 @@ class LocalToGlobalIndexMap final
         std::vector<MeshLib::Element*> const& elements,
         NumLib::MeshComponentMap&& mesh_component_map);
 
+    /// Private constructor used by internally created local-to-global index
+    /// maps. The mesh_component_map is passed as argument instead of being
+    /// created by the constructor.
+    /// \attention The passed mesh_component_map is in undefined state after
+    /// this construtor.
+    explicit LocalToGlobalIndexMap(
+        std::vector<std::unique_ptr<MeshLib::MeshSubsets>>&& mesh_subsets,
```

Comment:
Btw, just out of curiosity: Why is `MeshLib::MeshSubsets` used everywhere in combination with `std::unique_ptr`? Is it an abstract type? Otherwiese one could store and pass it a usual object or reference.